Objects/typeobject.c

```diff
@@ -9514,7 +9514,7 @@ super_init_without_args(_PyInterpreterFrame *cframe, PyCodeObject *co,
 
     // Look for __class__ in the free vars.
     PyTypeObject *type = NULL;
-    int i = co->co_nlocals + co->co_nplaincellvars;
+    int i = PyCode_GetFirstFree(co);
```

Comment:
Py_ssize_t?

Reply:
This should be an. I'll change `PyCode_GetFirstFree`.

Reply:
@markshannon why should it be `int`?